packages/Algernon.package/ALGQuitType.class/class/algertemList.st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+algertemList
+	
+	^ {ALGItem withAlgertype: (self new: 'save' commandBlock: [Smalltalk saveSession.]).
```

Comment:
Just for comparing the actions in code it would be more readable if you wrote `self snapshot: true andQuit: false`